src/lib/fountain/FountainMetadata.h

```diff
@@ -33,7 +33,7 @@ class FountainMetadata
 		to_uint8_arr(encode_id, size, d);
 	}
 
-	unsigned id() const
+	uint32_t id() const
```

Comment:
This unsigned was doing a lot of work -- the type here should've been 32bit all along.